cruisetrack/helper/__init__.py

```diff
@@ -0,0 +1,6 @@
+import pickle
```

Comment:
this will help us create test scenarios

pickle variables into a file, unpickle it later.